packages/eslint-plugin/src/rules/dependency-checks.ts

```diff
@@ -96,10 +96,10 @@ export default ESLintUtils.RuleCreator(
       },
     ]
   ) {
-    if (!(context.parserServices as any).isJSON) {
+    if (!(context.sourceCode.parserServices as any).isJSON) {
```

Comment:
`parserServices` was moved to the `context.sourceCode` and `context.parserServices` was deprecated